README.md

```diff
@@ -350,18 +350,71 @@ We expect to add Monero into the ports tree in the near future, which will aid i
 
 ### On OpenBSD:
 
-This has been tested on OpenBSD 5.8.
+This has been tested on OpenBSD 6.2.
 
-You will need to add a few packages to your system. `pkg_add db cmake gcc gcc-libs g++ miniupnpc gtest`.
+You will need to add a few packages to your system. Choose version 4 for db. `pkg_add db cmake miniupnpc zeromq`.
 
 The doxygen and graphviz packages are optional and require the xbase set.
 
-The Boost package has a bug that will prevent librpc.a from building correctly. In order to fix this, you will have to Build boost yourself from scratch. Follow the directions here (under "Building Boost"):
-https://github.com/bitcoin/bitcoin/blob/master/doc/build-openbsd.md
```

Comment:
That seems like something useful to know for 5.8, no ? Unless you believe it is actually incorrect, or it was fixed in boost 1.64 ?

Reply:
Not only that, can we put those directions in our own repo so we're responsible for them and not the bitcoin crew?

Reply:
As far as I know, it is not fixed by Boost.